scripts/mkcloud

```diff
@@ -474,6 +474,11 @@ function setuphost()
             libvirt $kvmpkg $osloader $ipxe lvm2 curl wget bridge-utils \
             dnsmasq netcat-openbsd ebtables libvirt-python
         [ "$?" == 0 -o "$?" == 4 ] || complain 10 "setuphost failed to install required packages"
+
+        # enable KVM
+        [[ $(uname -m) = s390x ]] && {
+            echo 1 > /proc/sys/vm/allocate_pgste
+        }
```

Comment:
As we have several places with such checks, maybe it makes sense to have a `is_arch` function.
But not necessarily as part of this PR. Could be refactored later.